cob_omni_drive_controller/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cob_omni_drive_controller)
 
+set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS}")
+
```

Comment:
@ipa-fxm `catkin_lint` will complain about this. Manually changing `CMAKE_CXX_FLAGS` should be avoided.

Use `add_compile_options(-std=c++11)` instead.

Reply:
thanks, @mgruhler for still caring about this repo
I knew it was something shorter ;-)

@pholthau could you make the according changes to the two affected pkgs?
I will then squash-merge...

Reply:
For sure! I still geht the notifications :-)